src/lib/util/EnvironmentVariable.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Builds expected env variables in this format:
+ * SFDX_SCANNER.<ENGINE>.VARIABLE_NAME
+ */
+
+import { ENGINE } from "../../Constants";
+
+const PREFIX = 'SFDX_SCANNER';
+const SEPARATOR = '_';
+
+export enum CONFIG_NAME {
+	MINIMUM_TOKENS = 'Minimum_Tokens'
+}
+
+function getEnvVariableName(engine: ENGINE, configName: CONFIG_NAME): string {
+	return `${PREFIX}${SEPARATOR}${engine.toUpperCase()}${SEPARATOR}${configName.toUpperCase()}`;
+}
+
+export function getEnvVariableAsString(engine: ENGINE, configName: CONFIG_NAME): string {
+	const envVariableName = getEnvVariableName(engine, configName);
+	return process.env[envVariableName];
+}
+
+export function getEnvVariableAsNumber(engine: ENGINE, configName: CONFIG_NAME): number {
```

Comment:
I think this will currently return NaN when the environment variable isn't set or isn't a number. 

Do we want it to return NaN if it's set, but not a number, but null if it isn't set? 

Whatever we choose, can you add documentation describing the contract when the value isn't set or the value can't be parsed?

Reply:
Good catch. We don't have a way to control what value the user may set. I see a suggestion on the forum to clean up the string before parsing:
```
parseInt(text.replace(/\D/g, ""),10);
```
where `\D` is a non-digit. Will add documentation for this as well.

Reply:
@jbartolotta-sfdc Let me know what you think about the fix.

Reply:
@rmohan20 - this change looks good.